File: jdroid-gradle-commons-plugin/src/main/java/com/jdroid/gradle/commons/versioning/IncrementPatchVersionTask.java
package com.jdroid.gradle.commons.versioning;

public class IncrementPatchVersionTask extends AbstractIncrementVersionTask {
	
	public IncrementPatchVersionTask() {
		setDescription("Increments the patch version (X.Y.Z) -> (X.Y.Z+1)");
	}

	@Override
	protected void incrementVersion(Version version) {
		version.incrementPatch();
	}

}

File: jdroid-gradle-commons-plugin/src/main/groovy/com/jdroid/gradle/commons/versioning/IncrementPatchVersionTask.kt
package com.jdroid.gradle.commons.versioning

open class IncrementPatchVersionTask : AbstractIncrementVersionTask() {

    init {
        description = "Increments the patch version (X.Y.Z) -> (X.Y.Z+1)"
    }

    override fun incrementVersion(version: Version) {
        version.incrementPatch()
    }
}
